packages/web/src/pages/sign-up-page/pages/SignUpPage.tsx

```diff
@@ -39,8 +39,7 @@ export const SignUpPage = (props: SignUpPageProps) => {
 
   const handleSubmit = useCallback(
     (values: SignUpEmailValues) => {
-      const { email } = values
-      onNext({ stage: 'create-password', params: { email } })
+      onNext({ stage: 'create-password' })
```

Comment:
@DejayJD I removed the email param because it's not needed here (we are using the Redux store for the form values). We should dispatch `dispatch(setValueField('email', values.email))` here

Reply:
great call